.cbom/cbom.json

```diff
@@ -0,0 +1,4100 @@
+{
```

Comment:
If this is documentation (?) what is the reason that this is not in the docs directory?

Reply:
Yes, I agree it can be moved to docs

Reply:
moved to docs folder